### ISSUE TITLE
Consider automatically adding 'svm' dependency

### ISSUE THREAD

**Issue by JakeWharton** [palantir/gradle-graal#377]
In order to help out Graal, I very often add
```groovy
compileOnly "org.graalvm.nativeimage:svm:20.1.0"
```
as a dependency.

Lately I've been doing
```groovy
compileOnly "org.graalvm.nativeimage:svm:${graal.graalVersion.get()}"
```
to avoid the split brain problem for the version specification.

It would be nice if this dependency was added automatically. Being `compileOnly` it doesn't affect downstream consumers like the normal application plugin output or a shadow jar or the like.

Just in case this has some negative interaction, a property could be added to disable this functionality.
```groovy
graal {
  svmDependency false
}
```
(or whatever you want to call it)

**Comment by yschimke** [comment]
Be careful with ordering here, if your dependencies are listed before the grall config, you get the default version.  Also I think you can drop the get(), I assume toString does the right thing.

**Comment by JakeWharton** [comment]
True. It probably would need applied in `afterEvaluate` to ensure the user has set their desired version.

**Comment by yschimke** [comment]
Also relevant you can easily trigger https://github.com/palantir/gradle-graal/issues/148 if you use implementation instead of compileOnly
